imports/ui/components/atoms/images/Logo.jsx

```diff
@@ -49,7 +85,19 @@ const LogoImage = styled.img`
   }
 `
 
-export const Logo = ({ image, size }) =>
-  image ? (
-    <LogoImage src={image.url} size={size} orientation={image.orientation} />
-  ) : null
+export const Logo = ({ text, size, color, font }) =>
+  text ? (
```

Comment:
In the original page the logic would be the other way around: If there is a logo image show the image otherwise use the text logo. Here, for the test project this is okay ;)